tekton/cache-upload.yaml

```diff
@@ -47,6 +47,11 @@ spec:
       description: |
         The path where to find the google credentials. If left empty, it is ignored.
       type: string
+      default: "/workspace/docker"
```

Comment:
btw, this shouldn't be on `googleCredentialsPath` param though (but on `docker-config`).

Reply:
Fixed